CHANGELOG.md

```diff
@@ -1,5 +1,22 @@
 # OverReact Test Changelog
 
+## 2.0.0
+
+__New Features__
+
+* Dart 2 compatible!
+
+__Breaking Changes__
+
+* The `getComponentPropKeys()` and `testPropForwarding()` functions have been
+  removed, as they depended on `dart:mirrors`. Once there is a Dart-2-only
+  release of `over_react`, this function will be re-added and will information
```

Comment:
Maybe a missing word here? "...and will depend information generated by the builder..."